great_expectations/rule_based_profiler/config/base.py

```diff
@@ -155,6 +155,7 @@ def __init__(
         name: str,
         class_name: str,
         module_name: Optional[str] = None,
+        evaluation_parameter_builder_configs: Optional[list] = None,
```

Comment:
Do we know what this `list` contains?

Reply:
@cdkini Yes, it contains `ParameterBuilder` configurations.  Since we did not expand this deeper in the signature of the constructor of the `RuleBasedProfilerConfig`, I did not do that here either.  We can do it (in a separate tiny PR).  Thanks.